package.json

```diff
@@ -12,11 +12,15 @@
   "dependencies": {
     "@types/highlight.js": "9.12.3",
     "@types/moment": "2.13.0",
+    "axios": "^0.21.0",
```

Comment:
Was it not possible to avoid this dependency?

Reply:
seems like I forgot to remove this, will do this